backend/src/graphql/arg/Paginated.ts

```diff
@@ -5,12 +5,12 @@ import { Order } from '@enum/Order'
 
 @ArgsType()
 export class Paginated {
-  @Field(() => Int, { nullable: true })
-  currentPage?: number
+  @Field(() => Int, { defaultValue: 1 })
+  currentPage: number
 
-  @Field(() => Int, { nullable: true })
-  pageSize?: number
+  @Field(() => Int, { defaultValue: 3 })
+  pageSize: number
 
-  @Field(() => Order, { nullable: true })
-  order?: Order
+  @Field(() => Order, { defaultValue: Order.DESC })
+  order: Order
```

Comment:
what does the new search feature "# in memo" has to do with pagination?
Is there a reason why to change the default settings of the class attributes?

Reply:
It's not directly related to pagination, although it does share that. However, I have refactored the Pagination object a bit along with the findContributions function to be a bit more object oriented. 
Instead of setting the default values in the three resolver functions that use Pagination, they are now only set in one place in the Paginated object.
That means the nullable fields were always filled with default values anyway.

Translated with www.DeepL.com/Translator (free version)